Sources/Moya/MultiTarget.swift

```diff
@@ -29,8 +29,8 @@ public enum MultiTarget: TargetType {
         return target.task
     }
 
-    public var validate: Bool {
-        return target.validate
+    public var validationType: ValidationType? {
+        return nil
```

Comment:
Maybe a `.none` case would be more expressive?

Reply:
Initially I had a `.none` case, but then I struggled with the ValidationType's `statusCodes` property: I didn't want it to return an empty array because if used wrongly it would cause Alamofire to error on every status code, so it would have to turn it into an optional. 

Secondly, in `MoyaProvider+Internal` It would meant much more "complex" code than a single line for both 3 methods with a `case let` to match the `.none`, an unwrapping of the newly optional `statusCode` property, and poor factorization capability because of the Alamofire's validation function being duplicated in DataRequest and DownloadRequest ( i.e 2 similar functions except for the generic type parameter).

As I see it, the nil version wins in term of code complexity. As for the usage, I admit being neutral.